Testscripts/Linux/TestRDMA_MultiVM.sh

```diff
@@ -126,8 +126,8 @@ function Run_IMB_MPI1() {
 				$mpi_run_path --allow-run-as-root --host $master,$slaves -n $(($mpi1_ppn * $total_virtual_machines)) $mpi_settings $imb_mpi1_path $extra_params > IMB-MPI1-AllNodes-output-Attempt-${attempt}.txt
 			;;
 			intel)
-				LogMsg "$mpi_run_path -hosts $master,$modified_slaves -ppn $mpi1_ppn -n $(($VM_Size * $total_virtual_machines)) $mpi_settings $extra_params"
-				$mpi_run_path -hosts $master,$modified_slaves -ppn $mpi1_ppn -n $(($VM_Size * $total_virtual_machines)) $mpi_settings $imb_mpi1_path $extra_params > IMB-MPI1-AllNodes-output-Attempt-${attempt}.txt
+				LogMsg "$mpi_run_path -hosts $master,$slaves -ppn $mpi1_ppn -n $(($VM_Size * $total_virtual_machines)) $mpi_settings $extra_params"
```

Comment:
revert please. this needs in IBM MPI case. they need separate machine name in param format.

Reply:
I don't understand, this is the case for Intel MPI, not affecting IBM MPI. If you check here, L510, https://github.com/LIS/LISAv2/blob/master/Testscripts/Linux/TestRDMA_MultiVM.sh#L510 , you'll see that IBM MPI is using indeed modified slaves, but every other MPI not (and modified_slaves variable is not available for any other MPI). It's a code defect, should have been $slaves from the beggining. This is actually what is fixing the Intel MPI run

Reply:
ok, you are right. This variable is only for IBM MPI